### PR TITLE
[12.0][IMP]partner_contact_address_default: removed field widget to allow filtering

### PR DESCRIPTION
**Pull request by antonio-trey**
Removed field selection widget in view to allowing filtering when "Allow all partner" option is checked.

### CODE REVIEW

**Review by MiguelPoyatos — APPROVED** [review]
LGTM!

**Review by rafadm98 — APPROVED** [review]
👍

File: partner_contact_address_default/README.rst [hunk 1]
```diff
@@ -66,6 +66,7 @@ Contributors
   * Sergio Teruel
 
 * Giovanni Serra <giovanni@gslab.it>
+* `Trey <https://www.trey.es>`_- Antonio González
```
Comment (rousseldenis) [inline partner_contact_address_default/README.rst]: Add yourself to readme/CONTRIBUTORS.rst instead

File: partner_contact_address_default/__manifest__.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 {
     'name': 'Partner Contact address default',
     'summary': 'Set a default delivery and invoice address for contacts',
-    'version': '12.0.1.2.1',
+    'version': '12.0.1.3.0',
```
Comment (rousseldenis) [inline partner_contact_address_default/__manifest__.py]: Usually you don't need to update this (maintainer will do it with ocabot) unless you have migration scripts

File: partner_contact_address_default/static/description/index.html [hunk 3]
```diff
@@ -415,6 +415,7 @@ <h2><a class="toc-backref" href="#id5">Contributors</a></h2>
 </ul>
 </li>
 <li>Giovanni Serra &lt;<a class="reference external" href="mailto:giovanni&#64;gslab.it">giovanni&#64;gslab.it</a>&gt;</li>
+<li><a class="reference external" href="https://www.trey.es">Trey</a> - Antonio González<ul>
```
Comment (rousseldenis) [inline partner_contact_address_default/static/description/index.html]: Remove this

**Review by rousseldenis — COMMENTED** [review]
Even if I understand the purpose, I want to have @chienandalu advice as it was introduced intentionally there https://github.com/OCA/partner-contact/commit/d075437ec126d04f199172e0ac7d0119d0e7326e

File: partner_contact_address_default/views/res_partner_views.xml [hunk 4]
```diff
@@ -11,14 +11,12 @@
                         <field
                             name="partner_delivery_id"
                             domain="[('id', 'child_of', commercial_partner_id), ('type', '=', 'delivery')]"
-                            widget="selection"
```
Comment (chienandalu) [inline partner_contact_address_default/views/res_partner_views.xml:14]: To keep the user experience, better use:
```xml
options="{'no_open': True, 'no_create': True}"
```

File: partner_contact_address_default/__manifest__.py (hunk 2 above)
Comment (antonio-trey) [reply partner_contact_address_default/__manifest__.py]: Done, thanks.

File: partner_contact_address_default/README.rst (hunk 1 above)
Comment (antonio-trey) [reply partner_contact_address_default/README.rst]: Done.

File: partner_contact_address_default/views/res_partner_views.xml (hunk 4 above)
Comment (antonio-trey) [reply partner_contact_address_default/views/res_partner_views.xml:14]: Thanks.

**Review by chienandalu — APPROVED** [review]
Thanks :)

**Comment by pedrobaeza** [comment]
Please fw-port it to upper versions

**Pull request merged**